src/views/components/top.pug

```diff
@@ -1,14 +1,57 @@
 .top-wrapper.js-section
+  //- script(type="application/javascript", async="async", src="https://icoholder.com/en/widget/big-green/21042.js?width=300")
```

Comment:
Why commented code is in PR? Should it be removed or uncommented?

Reply:
it is from task with widgets here. This widget wasnt working cause of icoholder.com crashing and i uncluded it but commented to use it after when u probably will ask to do it again